lib/formatters/junit.js

```diff
@@ -24,6 +25,16 @@ function getMessageType(message) {
 
 }
 
+/**
+ * Returns a full file path without extension
+ * @param {string} filePath input file path
+ * @returns {string} file path without extension
+ * @private
+ */
+function pathWithoutExt(filePath) {
+    return path.join(path.posix.dirname(filePath), path.basename(filePath, path.extname(filePath)));
```

Comment:
Should this be `path.posix.join`? Otherwise, it seems like on Windows, the path to the directory could have forward slashes while the filename would be preceded by a backslash.

Reply:
Thank you! I've made the suggested fix.